src/Aspire.CommunityToolkit.Hosting.Meilisearch/Aspire.CommunityToolkit.Hosting.Meilisearch.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AdditionalPackageTags>integration hosting meilisearch</AdditionalPackageTags>
+    <Description>Meilisearch support for .NET Aspire.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\VolumeNameGenerator.cs" Link="Utils\VolumeNameGenerator.cs" />
+    <!--<Compile Include="$(ComponentsDir)Aspire.Meilisearch\MeilisearchHealthCheck.cs" Link="MeilisearchHealthCheck.cs"></Compile>-->    
```

Comment:
Is this blocked until aspire-9?

Reply:
Just the health check part needs aspire-9. but it's not necessary for now. We can add a health check to hosting at https://github.com/CommunityToolkit/Aspire/issues/73